### PR TITLE
ORFS Flow Tutorial test scripts added

### PR DESCRIPTION
**Pull request by vijayank88**
Signed-off-by: vijayank88 <paruthi143@gmail.com>
This PR is part of https://github.com/The-OpenROAD-Project/OpenROAD/pull/1559
Updated README.md and added tutorials/TutorialsScripts as per directory structure suggested by @vvbandeira

### CODE REVIEW

File: flow/tutorials/README.md
Comment (vvbandeira) [inline flow/tutorials/README.md]: users

File: flow/tutorials/README.md
Comment (vvbandeira) [inline flow/tutorials/README.md]: Refer to the

File: flow/tutorials/README.md
Comment (vvbandeira) [inline flow/tutorials/README.md]: Let's wait for the OR PR and use the readthedocs link.

File: flow/tutorials/README.md
Comment (vvbandeira) [inline flow/tutorials/README.md]: The tutorial uses sample test scripts already part of OpenROAD `test`.

File: flow/tutorials/README.md
Comment (vvbandeira) [inline flow/tutorials/README.md]: `flow/tutorials/TutorialScripts` feels redundant, `flow/tutorials/scripts` would be better or even under the existing scripts folder `flow/scripts/tutorials`
any thoughts @maliberty / @rovinski?

**Review by vvbandeira — CHANGES_REQUESTED** [review]

File: flow/tutorials/README.md
Comment (vvbandeira) [reply flow/tutorials/README.md]: also, @vijayank88 convert to relative links since this is in the same repo

Comment (vijayank88) [reply flow/tutorials/README.md]: @vvbandeira I have updated the README.md based on your review. Once OR PR approved, then I will push my updates.

Comment (rovinski) [reply flow/tutorials/README.md]: I prefer `flow/tutorials/scripts`.

File: flow/tutorials/TutorialScripts/drt/drc_fix.tcl
Comment (rovinski) [inline flow/tutorials/TutorialScripts/drt/drc_fix.tcl]: Why hard-coded to 2?

File: flow/tutorials/TutorialScripts/drt/drc_fix.tcl
Comment (rovinski) [inline flow/tutorials/TutorialScripts/drt/drc_fix.tcl]: Why do detailed route and then diff the guide file? Why are results being compared instead of simply checking if `[detailed_route_num_drvs] != 0`.

File: flow/tutorials/TutorialScripts/drt/drc_issue.tcl
Comment (rovinski) [inline flow/tutorials/TutorialScripts/drt/drc_issue.tcl]: Same questions

File: flow/tutorials/README.md
Comment (vijayank88) [reply flow/tutorials/README.md]: @rovinski We've already flow/scripts. TutorialScripts kind of unit test for new users.

File: flow/tutorials/TutorialScripts/drt/drc_fix.tcl
Comment (vijayank88) [reply flow/tutorials/TutorialScripts/drt/drc_fix.tcl]: @rovinski Its a simple gcd design, take less then a min to complete. User can change as needed.

File: flow/tutorials/README.md
Comment (dralabeing) [reply flow/tutorials/README.md]: @vijayank88 Please change the subdirectory name to scripts ..since this is under flow/tutorials

File: flow/tutorials/README.md
Comment (dralabeing) [reply flow/tutorials/README.md]: @vijayank88 I agree Once the OR PR has been approved, make this final change

File: flow/tutorials/TutorialScripts/drt/drc_fix.tcl
Comment (rovinski) [reply flow/tutorials/TutorialScripts/drt/drc_fix.tcl]: If it's copied then why can't it be linked or symlinked?

File: flow/tutorials/TutorialScripts/drt/drc_fix.tcl
Comment (rovinski) [inline flow/tutorials/TutorialScripts/drt/drc_fix.tcl]: Hardcoding 32 is definitely not the right answer here. A vast majority of users don't have that many threads.

Comment (vijayank88) [reply flow/tutorials/TutorialScripts/drt/drc_fix.tcl]: @rovinski Yes updated as follows
```
#2 CORES enough to run this design, change if needed
set_thread_count 2
```

File: flow/tutorials/README.md
Comment (vvbandeira) [reply flow/tutorials/README.md]: @vijayank88 please update this with https://openroad.readthedocs.io/en/latest/tutorials/FlowTutorial.html and add a period at the end of all sentences/ paragraphs.

**Review by vvbandeira — APPROVED** [review]
Looks good to me. @rovinski any last comments?

**Comment by rovinski** [comment]
I don't like the copying of files from OR but I'm not going to hold up the PR.

**Pull request merged**
